tests/settings.py

```diff
@@ -3,7 +3,8 @@
 BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
 SECRET_KEY = '7&cb7ybp)-z@f5ow8jryz=0*b!@4ma%e#bl2$z!+_g!i3*8=k_'
 DEBUG = True
-ALLOWED_HOSTS = ['localhost']
+ALLOWED_HOSTS = ['localhost',
```

Comment:
Try `ALLOWED_HOSTS = []`, in development mode (when `DEBUG` is `True`), this should allow both `localhost` and `127.0.0.1`, I just tried this now and it works.

Reply:
Ok. I'll check that out and modify the commit :smile:

Reply:
if you look at the other openwisp modules, they all use `ALLOWED_HOSTS = []`, which is the default created by django when you create a new project, so it's better to maintain consistency.

Reply:
Oh... Got it. I'll change :smile:

Reply:
Done.. tell if changes needed